codes/python/monitor.py

```diff
@@ -18,29 +18,47 @@ def get_disk_space():
     pass
 
 
-def backup(sub_files):
+def backup(sub_files, own=False):
     # files = ["gps_binex.log", "weather_data.log", "thermostat.log", "solar.log", ]
-    source = "/media/mmcblk0p1/logs"
+    time_now = datetime.datetime.now()
+    time_now = str(time_now.year) + "_" + str(time_now.month) + "_" + \
+        str(time_now.day) + "_" + str(time_now.hour) + str(time_now.minute)
```

Comment:
There is a built in function for date formatting: `time_now.strftime('%Y_%m_%d_%H_%M_%S')`